gateway-inbound copy.Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM asecurityteam/serverfull-gateway
+COPY api-inbound.yaml .
```

Comment:
Nit: I'd recommend renaming these to `gateway-inbound.yaml`/`gateway-outbound.yaml` so the YAML config and Dockerfile are right next to each other.